definitions/ext-rf_scanner/dashboard.json

```diff
@@ -0,0 +1,167 @@
+{
+  "name": "Golden_Metrics_rf-device",
+  "description": null,
+  "permissions": "PUBLIC_READ_WRITE",
+  "pages": [
+    {
+      "name": "Golden_Metrics_rf-device",
+      "description": null,
+      "widgets": [
+        {
+          "visualization": {
+            "id": "viz.bar"
+          },
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "height": 3,
+            "width": 4
+          },
+          "title": "Total Time",
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 2811838,
+                "query": "SELECT average(totalTime) FROM THD_RF_TRANSACTION FACET macAddress"
+              }
+            ]
+          },
+          "linkedEntityGuids": [
+            "MjgxMTgzOHxWSVp8REFTSEJPQVJEfDE4MjM2MDg"
+          ]
+        },
+        {
+          "visualization": {
+            "id": "viz.bar"
+          },
+          "layout": {
+            "column": 5,
+            "row": 1,
+            "height": 3,
+            "width": 4
+          },
+          "title": "Actions",
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 2811838,
```

Comment:
We should anonymize this, could you set the account ID to 0 for example? thank you!